nats_test.go

```diff
@@ -98,6 +98,94 @@ func TestVersionMatchesTag(t *testing.T) {
 	}
 }
 
+func TestExpandTildePath(t *testing.T) {
+	t.Run("unix", func(t *testing.T) {
+		if runtime.GOOS == "windows" {
+			t.Skip()
```

Comment:
Let's see if we can get one of these from @ColinSullivan1

Reply:
Will do...